### PR TITLE
Combine Search Audit Index articles

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) merges the [Log Search Audit Index](https://help.sumologic.com/docs/manage/security/log-search-audit-index/) article into this article:
https://help.sumologic.com/docs/manage/security/search-audit-index/

Issue number: GitHub issue #2990

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
Could you give this a look, @wjakelee, since this PR provides the fix for the issue you reported (#2990)?

File: docs/manage/security/search-audit-index.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
 id: search-audit-index
 title: Search Audit Index
-description: Learn how to enable the Search Audit Index.
+description: The Search Audit Index provides event logs on search usage and activities for your account. 
 ---
 
+The Search Audit Index provides event logs on search usage and activities for your account. The index allows you to monitor and audit the search queries being run within your account, the types of queries, the users running them, and more. The [Enterprise Search Audit App](/docs/integrations/sumo-apps/enterprise-search-audit) provides pre-built dashboards and reports of the data from the Search Audit Index to help you analyze your current search use and identify areas for improvement.
```
Comment (kimsauce) [inline docs/manage/security/search-audit-index.md:7]:
```suggestion
The Search Audit Index provides event logs on search usage and activities for your account. The index allows you to monitor and audit the search queries being run within your account, the types of queries, the users running them, and more. 

To visualize your Search Audit Index data, download the [Enterprise Search Audit App](/docs/integrations/sumo-apps/enterprise-search-audit), which provides pre-built dashboards and reports to help you analyze your current search use and identify areas for improvement.
```

**Review by wjakelee — APPROVED** [review]
Looks Good to me!

**Review by kimsauce — CHANGES_REQUESTED** [review]
one suggestion

**Review by kimsauce — APPROVED** [review]

**Pull request merged**